R/registry.R

```diff
@@ -163,7 +163,30 @@ get_cran_archived <- function() {
   w <- tibble::as_tibble(jsonlite::fromJSON(z$parse("UTF-8"))$package)
   dplyr::select(w, .data$Package, .data$Type)
 }
-is_staff <- is_cran_archived <- function(x, y) x %in% y
+is_cran_archived <- function(x, y) x %in% y
+
+is_staff <- function(maintainer, pkg_name, staff, folder = folder) {
+  # from pkgdown
+  path_first_existing <- function(...) {
+  paths <- fs::path(...)
+  for (path in paths) {
+    if (fs::file_exists(path))
+      return(path)
+  }
+
+  NULL
+}
+
+  path <- path_first_existing(paste0(dir(folder, full.names = TRUE), "/", pkg_name))
```

Comment:
it's a bit stupid but at this point of the pipeline we have the codemeta.json with no trace of the folder where the package was, so we have to re-find it.